torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -163,6 +163,70 @@ bool validateKernelArg(
   }
 }
 
+// Return true if all the tensors have the same stride
+bool checkSameStride(const std::vector<c10::IValue>& tensors) {
+  for (size_t idx = 0; idx < tensors.size() - 1; ++idx) {
```

Comment:
This seems to be potentially erroneous. Please make sure `tensors.size()` is greater than zero.

Reply:
Actually, C++ tests seem to be failing due to this. I'll fix this.

Reply:
Fixed by 9630219aed14c23ac80a02097d0dca65ac6bb50c